TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md

```diff
@@ -0,0 +1,130 @@
+* **Declared project dependencies**: [Google Sheets](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0)
+* **CCC YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+
+## Agenda and Minutes
+
+### Welcome
+   * Dave Thaler (DT) opened the call at 7:05am PT
+   * DT welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
+   * Kurt Taylor (KT) captured the meeting minutes
+   * DT reviewed the agenda and asked for any additions
+
+### Roll Call / Attendance
+
+***Voting members of the TAC***
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+   * Accenture - Giuseppe Giordano
+   * Ant Group - Hongliang Tian (Tate)
+   * Arm - Thomas Fossati + / Michael Lu
+   * Google - Cfir Cohen  / Catalin Sandu +
+   * Huawei - Zhipeng (Howard) Huang +
+   * Intel - Dan Middleton + / Simon Johnson
+   * Meta - Eric Northup + / Shankaran Gnanashanmugam
+   * Microsoft - Dave Thaler (TAC chair) +
+   * Red Hat - Lily Sturmann +  / Dimitrios Pendarakis
+" + " indicates attending
+
+***Other Attendees***
+
+   * Jordan Brandt (Inpher)
+   * Mike Bursell (Profian)
+   * XiaoFeng Wang (Indiana University)
+   * Alec Fernandez (Microsoft)
+   * Eric Voit (Cisco)
+   * Francois-Xavier Marseille (THALES)
+   * Jethro Beekman (Fortanix)
+   * Matthieu Legre (Cysec)
+   * Naveen Cherukuri (NVIDIA)
+   * Nick Vidal (Profian)
+   * Penglin Yang (China Mobile)
+   * Steve Van Lare (Anjuna)
+   * Mark Novak (JP Morgan Chase)
+   * Mona Vij (Intel)
+   * Xinxin Fan (IoTex)
+   * Ahmad Atamli (NVIDIA)
+   * Hesham ElBakoury (Consultant)
+   * Ravi Sahita (Rivos)
+   * Drasko Draskovic (Ultraviolet)
+   * Ananya Garg (Microsoft)
+   * Helen Lau (Linux Foundation)
+   * Kurt Taylor (Linux Foundation)
+
+
+**Meeting minute approvals**
+
+Proposed: That the minutes of the September 8, 2022 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+   * Quorum was reached and the minutes were approved by the voting members of the TAC. Catalin Sandu abstained.
+
+
+**TAC Tech Talks**
+
+   * NSF Center for Distributed Confidential Computing (CDCC) - Prof. Xiaofeng Wang
+       * Mark Novak - Possible collaboration with new Governance (GRC) SIG
```

Comment:
What is above line supposed to mean?   Is there an action item?

Reply:
Mark stated that it was just a comment